cicd/pipelines/remote-pipelines-tasks-resolvers.adoc

```diff
@@ -0,0 +1,52 @@
+
+toc::[]
+
+Pipelines and tasks are reusable blocks for your CI/CD processes. You can reuse pipelines or tasks that you previously developed, or that were developed by others, without having to copy and paste their definitions. These pipelines or tasks can be available from several types of sources, from other namespaces on your cluster to public catalogs.
+
+In a pipeline run resource, you can specify a pipeline from an existing source. In a pipeline resource or a task run resource, you can specify a task from an existing source.
+
+In these cases, the _resolvers_ in {pipelines-title} retrieve the pipeline or task definition from the specified source at run time.
+
+The following resolvers are available in a default installaton of {pipelines-title}:
+
+* Hub resolver:: retrieves a task or pipeline from the Pipelines Catalog available on {artifact-hub} or {tekton-hub}.
+* Bundles resolver:: retrieves a task or pipeline from a Tekton bundle, which is an OCI image available from any OCI repository, such as an OpenShift container repository.
+* Cluster resolver:: retrieves a task or pipeline that is already created on the same {product-title} cluster in a specific namespace.
+* Git resolver:: retrieves a task or pipeline binding from a Git repository. You must specify the repository, the branch, and the path.
+
+[id="resolver-hub_{context}"]
+== Specifying a remote pipeline or task from a Tekton catalog
+You can specify a remote pipeline or task that is defined in a public Tekton catalog, either link:https://artifacthub.io/[{artifact-hub}] or link:https://hub.tekton.dev/[{tekton-hub}] by using the Hub resolver.
+
+include::modules/op-resolver-hub-config.adoc[leveloffset=+2]
+include::modules/op-resolver-hub.adoc[leveloffset=+2]
+
+[id="resolver-bundles_{context}"]
+== Specifying a remote pipeline or task from a Tekton bundle
+
+You can specify a remote pipeline or task from a Tekton bundle by using the Bundles resolver. A Tekton bundle is an OCI image available from any OCI repository, such as an OpenShift container repository.
+
+include::modules/op-resolver-bundle-config.adoc[leveloffset=+2]
+include::modules/op-resolver-bundle.adoc[leveloffset=+2]
+
+[id="resolver-cluster_{context}"]
+== Specifying a remote pipeline or task from the same cluster
+
+You can specify a remote pipeline or task that is defined in a namespace on the {product-title} cluster where {pipelines-title} is running by using the Cluster resolver.
+
+include::modules/op-resolver-cluster-config.adoc[leveloffset=+2]
+include::modules/op-resolver-cluster.adoc[leveloffset=+2]
+
+[id="resolver-git_{context}"]
+== Specifying a remote pipeline or task from a Git repository
+
+You can specify a remote pipeline or task from a Git repostory by using the Git resolver. The repository must contain a YAML file that defines the pipeline or task. The Git resolver can access a repository either by cloning it anonymously or else by using the authenticated SCM API.
+
```

Comment:
These say "by using..." when referring to the resolvers, but the modules have titles like "Configuring the bundles resolver" - I'd go with "configuring" > "using" consistently

Reply:
"configuring" is only a part of "using".